### PR TITLE
fix: support stories config with configuration objects

### PR DESCRIPTION
**Pull request by stevoland**
Issue: https://github.com/storybookjs/react-native/issues/423 sb-rn-get-stories does not support stories config with a configuration object (titlePrefix)

## What I did

Updated loader script to use [`normaliseStories`](https://github.com/storybookjs/storybook/blob/next/code/lib/core-common/src/utils/normalize-stories.ts) which supports string globs and configuration objects

## How to test

Use configuration objects and remove `title` property from story metadata.

```js
module.exports = {
  stories: [
    {
      files: '**/*.stories.tsx',
      directory: './components',
      titlePrefix: 'ComponentsPrefix',
    },
  ],
```

### CODE REVIEW

File: app/react-native/scripts/__snapshots__/loader.test.js.snap [hunk 1]
```diff
@@ -91,7 +140,7 @@ import \\"@storybook/addon-ondevice-actions/register\\";
     
 
       const getStories=() => {
-        return {\\"./scripts/mocks/exclude-config-files/include-components/FakeStory.stories.tsx\\": require(\\"include-components/FakeStory.stories.tsx\\")};
+        return {\\"./scripts/mocks/exclude-config-files/include-components/FakeStory.stories.tsx\\": require(\\"./include-components/FakeStory.stories.tsx\\")};
```
Comment (stevoland) [inline app/react-native/scripts/__snapshots__/loader.test.js.snap:143]: I've changed this snapshot but it seems correct now?

**Comment by dannyhw** [comment]
@stevoland thanks for your contribution 🙏. This looks really great!

 I'll go through this more thoroughly soon, sorry for the delay I've been very busy unfortunately.

**Comment by stevoland** [comment]
> I'll go through this more thoroughly soon, sorry for the delay I've been very busy unfortunately

Thanks, no rush. I know the feeling!

**Review by dannyhw — APPROVED** [review]

**Comment by dannyhw** [comment]
Looks good, thanks again!

**Pull request merged**